recipes/clapper/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "clapper" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/clapper-{{ version }}.tar.gz
+  sha256: 31494e3cd7c2ccf1b5884e382d75429e24da2aee1e19f07b0508c95a7ab772eb
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools >=61.0.0
+    - wheel
+    - pip
+  run:
+    - python >=3.9
+    - click >=8
+    - tomli
+    - tomli-w
+    - xdg
```

```suggestion
    - xdg
    - __unix
```
This virtual package makes sure that your package can only be installed on linux and osx. 

Ref: https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-virtual.html

---

This is nice to know, thanks!